vllm/config/scheduler.py

```diff
@@ -37,6 +37,11 @@ class SchedulerConfig:
     This config has no static default. If left unspecified by the user, it will
     be set in `EngineArgs.create_engine_config` based on the usage context."""
 
+    prefill_max_num_batched_tokens: SkipValidation[int] = None
```

Comment:
Why `SkipValidation`? If it's because this will be set later by us if `None` you can do:

```suggestion
    prefill_max_num_batched_tokens: int = Field(default=None)
```

This will skip validateion for the default `None` but validate passed values.